cclib/parser/fchkparser.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Parser for Formatted Checkpoint files"""
+
+
+from __future__ import print_function
+
+import re
+
+import numpy
+
+from cclib.parser import data
+from cclib.parser import logfileparser
+from cclib.parser import utils
+
+SHELL_ORBITALS = {
+    0: ['S'],
+    1: ['PX', 'PY', 'PZ'],
+    -1: ['S', 'PX', 'PY', 'PZ'],
+    2: ['D1', 'D2', 'D3', 'D4', 'D5', 'D6'],
+    -2: ['D1', 'D2', 'D3', 'D4', 'D5'],
+    3:  ['F1', 'F2', 'F3', 'F4', 'F5', 'F6', 'F7', 'F8', 'F9', 'F10'],
+    -3: ['F1', 'F2', 'F3', 'F4', 'F5', 'F6', 'F7'],
+    4: ['G1', 'G2', 'G3', 'G4', 'G5', 'G6', 'G7', 'G8', 'G9', 'G10', 'G11', 'G12','G13'],
+    -4: ['G1', 'G2', 'G3', 'G4', 'G5', 'G6', 'G7', 'G8', 'G9']
+
+}
+
+SHELL_START = {
+    0: 1,
+    1: 2,
+    -1: 2,
+    2: 3,
+    -2: 3,
+    3: 4,
+    -3: 4
+}
+
+
+def shell_to_orbitals(type, offset):
```

Comment:
Could you add a docstring?

Perhaps also prefix with underscore - this is meant to be private to the module right?

Reply:
Done.